LocalPackages/PreferencesUI/Sources/PreferencesUI/View+roundedBorder.swift

```diff
@@ -1,7 +1,7 @@
 //
-//  RoundedBorder.swift
+//  View+roundedBorder.swift
```

Comment:
This file was duplicated in a few places and I'm now moving it to `PreferencesUI`.